mosip-identity-plugin/src/main/java/io/mosip/esignet/plugin/mosipid/dto/IdaKycAuthRequest.java

```diff
@@ -29,7 +31,7 @@ public class IdaKycAuthRequest {
     private String requestSessionKey;
     private Map<String, Object> metadata;
     private List<String> allowedKycAttributes;
-
+    private Boolean claimMetadataRequired;
     @Data
```

Comment:
Please provide on new line gap between L34 and L35

Reply:
Provided gap now